portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -8,6 +8,7 @@
   private long id;
   private String nickname;
   private String text;
+  private float sentiment;
```

Comment:
[This site](https://javarevisited.blogspot.com/2016/05/difference-between-float-and-double-in-java.html#:~:text=Difference%20between%20float%20and%20double%20variable%20in%20Java%3F,6%20to%207%20decimal%20digits.) says that doubles are more precise than floats in java, if it matters.

Reply:
The sentiment value returned by the API has datatype float, so I think it doesn't matter if I use a double since the value returned by the API will have the precision of a float anyways.